cluster-provision/k8s/1.21/manifests/prometheus/grafana/grafana-dashboardDefinitions.yaml

```diff
@@ -1,7 +1,2206 @@
 apiVersion: v1
 items:
 - apiVersion: v1
-  data:
+  data:    
+    kubevirt-control-plane.json: |-
```

Comment:
Since we already have a grafana dashboard manifest here in this 1.21 specific directory tree, i don't see any issue with adding the control plane dashboard. Maybe in the future we'll want to consider moving this out to the `cluster-provision/manifests/` dir when new CI providers are createad?

Reply:
@marceloamaral pointed out that we have a dashboard git repo too. Maybe it makes sense to move it in a next step there and just pull in the dashboards on `cluster-up`. That should give us faster iterations.